app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ow-mod-manager",
   "productName": "OuterWildsModManager",
-  "version": "1.10.1",
+  "version": "1.11.0",
```

Comment:
revert the version change, I keep version changes in dedicated commits. Makes it easier to avoid conflicts and to track version changes in the commit history.